pkgs/by-name/ff/ffmpeg-livepeer/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  ffmpeg_7-headless,
+  lib,
+  fetchFromGitHub,
+  apple-sdk_11,
+  stdenv,
+  clang,
+}:
+
+let
+  version = "7.0.1-unstable-2024-07-10";
+in
+
+(ffmpeg_7-headless.override {
```

Comment:
Nit: It's not strictly necessary to use the versioned ffmpeg here because you're overriding the version anyways; it doesn't make a difference.